<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-OO0DI5VI" failures="0" tests="6" name="testCases.TC003_LoginDDT" time="0.012" errors="1" timestamp="2025-08-19T00:32:27 IST" skipped="5">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="setup" time="0.012">
    <error message="Cannot invoke &quot;String.equalsIgnoreCase(String)&quot; because the return value of &quot;java.util.Properties.getProperty(String)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because the return value of "java.util.Properties.getProperty(String)" is null
at testBase.BaseClass.setup(BaseClass.java:58)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="0.000">
    <skipped/>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- testCases.TC003_LoginDDT -->
